release-calendar/src/server/repository-service.ts

```diff
@@ -43,14 +43,9 @@ export class RepositoryService {
       .createQueryBuilder('release')
       .leftJoinAndSelect('release.promotions', 'promotion')
       .where('release.name = :name', {name})
+      .orderBy('promotion.date', 'DESC')
```

Comment:
Oh this works?

Reply:
Yep!

Reply:
Nice :)

Reply:
We need to start writing unit tests soon...